Makefile

```
@@ -0,0 +1,35 @@
+PKG_SCRIPTS=scripts
+
+default: help
```

checkmake (0.2.2)

[warning] Target "default" should be declared PHONY.

(phonydeclared)